manifest/manifest.go

```diff
@@ -147,7 +151,7 @@ func GuessMIMEType(manifest []byte) string {
 			}
 			return ociMan.Config.MediaType
```

Comment:
(Just noticed: This doesn’t make sense to me — it is mixing config and manifest media type, and an index should not have a config field. Obviously unrelated.)

Reply:
That also jumped in my eyes but I assumed that I missed the point.  Thanks for highlighting!